maestro-android/src/main/java/dev/mobile/maestro/handlers/AbstractSettingHandler.kt

```diff
@@ -0,0 +1,42 @@
+package dev.mobile.maestro.handlers
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.util.Log
+
+abstract class AbstractSettingHandler(private val context: Context, private val permissions: List<String>) {
```

Comment:
Let's include this code to LocationSettingHandler nobody else is using it.

Reply:
I thought it makes sense to keep it in the separate file, if we'll have more setting handlers in the future, it'd be easy to reuse this logic